README.adoc

```diff
@@ -57,12 +57,18 @@ Try to use at least version `0.11.0` as we haven't tested another version.
 Assuming you have Go installed and configured (have `$GOPATH` setup) here is
 how to build.
 
-Check out the code
+Check out the code. On Linux or Mac:
 
 ----
 $ git clone https://github.com/almighty/almighty-core $GOPATH/src/github.com/almighty/almighty-core
 ----
 
+On Windows PowerShell:
+
+----
+$ git clone https://github.com/almighty/almighty-core $env:GOPATH\src\github.com\almighty\almighty-core
+----
```

Comment:
Ok, let me know how you want me to handle this then. I can file an issue
for all of them if you prefer.

On 2 August 2016 at 17:41, Max Rydahl Andersen <notifications@github.com>
wrote:

> In README.adoc
> <https://github.com/almighty/almighty-core/pull/75#discussion_r73191591>:
>
> >
> >  ----
> >  $ git clone https://github.com/almighty/almighty-core $GOPATH/src/github.com/almighty/almighty-core
> >  ----
> >
> > +On Windows PowerShell:
> > +
> > +----
> > +$ git clone https://github.com/almighty/almighty-core $env:GOPATH\src\github.com\almighty\almighty-core
> > +----
>
> there are about 12 other references to $GOPATH in the readme....need to
> find a better way to avoid having to repeat this kind of lines 12 times ;/
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/almighty/almighty-core/pull/75/files/b980ffe14668c3c442f9a7d4178f6e51d3c2a5e4#r73191591>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAJoQcWiSRuUHRt40hfR8L4poFq19d5fks5qb3M9gaJpZM4Jazl3>
> .
>